apps/api-extractor/src/api/Extractor.ts

```diff
@@ -285,100 +286,23 @@ export class Extractor {
       });
     }
 
-    let apiReportChanged: boolean = false;
```

Comment:
Since we now (potentially) generate multiple report files, this logic was extracted into a separate static method.